### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by jmburges**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Cookes and Sessions
+
+## Objective
+
+We're going to make a very simple shopping cart, stored entirely in the session.
+
+## Introduction
+
+The Rails `session` method gives us access to the Rails session. The session is a datastore implemented with cookies. You can store simple data structures in the session. ActiveRecord models, no. Arrays of strings or numbers, yes. Basically, if it could be converted into JSON, it can go in the `session`.
```
Comment (jmburges) [inline README.md]: They don't quite know what JSON is. Maybe just say keep it to hashes, arrays, strings and numbers

File: test/controllers/application_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```
Comment (jmburges) [inline test/controllers/application_controller_test.rb]: So we actually use Rspec for all of our testing. Rspec and Capybara for the more view layer stuff. Take a look at this: https://github.com/learn-co-curriculum/formal-affair-rails/blob/399de77968a7adae2ff1212f95e70ed5028e2895/spec/concrete_form_spec.rb for an idea of what our specs look like

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Instructions
+
+You'll find that the rspec is failing. Get them to pass. I would tackle it like this:
+
+1. `ApplicationController#cart`s tests are failing because, primarily, it doesn't exist. Write it to make them pass.
+2. Now `rails generate` a Products controller with two actions, index and add.
+3. Fill them in to make the specs pass.
+
+After the specs pass, you'll find you still don't have an app, for you have no views. Rails will tell you which ones you need to write.
```
Comment (jmburges) [inline README.md]: I would include some capybara tests to solve this issue. Somethign like this https://github.com/learn-co-curriculum/rails-pollywog-ranch/blob/5155f9d5137794692d36d50a188607f0c424e0c3/spec/03_features/metamorphosize_spec.rb#L29-L33

**Comment by jmburges** [comment]
Biggest change is to add the capybara and change to rspec style tests

**Comment by jmburges** [comment]
but back at you!

**Comment by queerviolet** [comment]
PTAL

**Pull request merged**